src/xcode/ENA/ENA/Resources/Localization/bg.lproj/privacy-policy.html

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
-<html lang="bg">
+<html lang="en">
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 
 		<title>Privacy Notice &bull; Corona-Warn-App</title>
-		<link rel="stylesheet" href="../style.css">
```

Comment:
Nope!

Reply:
![Simulator Screen Shot - iPhone 12 Pro - 2021-07-13 at 11 15 07](https://user-images.githubusercontent.com/149976/125425827-5f4d72ca-9d1e-4265-98e2-4023eac8e739.png)

Removing the stylesheet would break the design.

Reply:
👍